docs/src/pages/TypesettingPage/TypesettingPage.js

```diff
@@ -108,13 +108,15 @@ const components = [
   {
     id: 'line-heights',
     title: 'Line height',
-    tokenMap: getPlatformTokens(
-      TOKENS,
-      IOS_TOKENS,
-      ANDROID_TOKENS,
-      ({ name, category }) =>
-        category === 'typesettings' && name.indexOf('LINE_HEIGHT') === 0,
-    ),
+    tokenMap: {
+      web: getPlatformTokens(
+        TOKENS,
+        IOS_TOKENS,
```

Comment:
We shouldn't be getting IOS or Android tokens here? Can we change `getPlatformTokens` to take IOS and Android Tokens optionally?

Reply:
Probably yeah